.goreleaser.yml

```diff
@@ -20,19 +19,19 @@ builds:
     ldflags:
       - -s -w -X main.commit={{.Commit}} -X main.date={{ .CommitDate }} -X github.com/cosmos/cosmos-sdk/version.Name=umee -X github.com/cosmos/cosmos-sdk/version.AppName=umeed -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }} -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger -X github.com/tendermint/tendermint/version.TMCoreSemVer={{ .Env.TM_VERSION }}
     goos:
-      - darwin
       - linux
     goarch:
       - amd64
-      - arm64
+#      - arm64  # github only supports linux @ amd64
 
 archives:
-  - id: bins
-    format: binary
-    name_template: "{{ .Binary }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+  # - id: bins
+  #   format: binary
+  #   name_template: "{{ .Binary }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
```

Comment:
@RafilxTenfen since we already have tar.gz archives, there is not much value in adding unpacked copies.

Reply:
IMO we should let the binaries and on the proposals we should point directly to the binaries
cc @umee-network/core-devs

Reply:
why?

Reply:
why not? 
kkk the binaries are directly what they want, we can set the checksum, and also cosmovisor works easier with the binaries instead of the archives

With the archives need to set up one env variable more. So, what is the reason for making it more complex than it needs to be

Reply:
Because it's a copy.

However, if validators will find it more useful then probably we should keep only binary. I will ask them for opinion.

Reply:
the archives are a copy wrapped in an tarball

Reply:
I know ;)

Reply:
Majority (40:10) want's binary, so will update to provide binary.
https://discord.com/channels/815699313581490196/940283450089435216/1044412287676592178